packages/web/docs/src/components/product-updates.tsx

```diff
@@ -38,21 +38,25 @@ export async function getChangelogs(): Promise<Changelog[]> {
   return productUpdatesFolder
     .slice(1) // cut `_meta.ts` which always comes first
     .map(item => {
-      if (!item.children) {
+      if (!('children' in item)) {
```

Comment:
The `.mjs` import now infers better type for the page map, and some constituents of the union don't have `children`, so we need to use `in` operator to check for it.